spec/models/application_gcse_qualification_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationGcseQualification, type: :model do
+  describe 'a valid GCSE qualification' do
+    it { is_expected.to validate_presence_of :qualification_type }
+    it { is_expected.to validate_presence_of :subject }
+    it { is_expected.to validate_presence_of :grade }
+    it { is_expected.to validate_presence_of :award_year }
```

Comment:
See above — we do collect institution details for GCSEs when they are non-UK

Reply:
We've split `institution_details` into institution_name and institution_country now.